zipline/assets/asset_writer.py

```diff
@@ -373,8 +457,16 @@ def init_db(self, engine):
                 primary_key=True),
             sa.Column('asset_type', sa.Text),
         )
+        self.version_table = _version_table_schema(metadata)
         # Create the SQL tables if they do not already exist.
         metadata.create_all(checkfirst=True)
+
+        # Check if the version is mismatched or, if it is not present, add it
+        version_found = check_version_info(self.version_table,
+                                           ASSET_DB_VERSION)
+        if not version_found:
+            write_version_info(self.version_table, ASSET_DB_VERSION)
```

Comment:
Could it be the case that we've got an old assets.db with no embedded version info, we then call `init_db`, which doesn't create anything because we use `checkfirst=True`, and then we insert the latest db version?  Seems like we should only write the version info on first creation of the all the tables, not even just the version table (since `create_all` will create any that are missing)?